src/main/java/service/RacingCars.java

```diff
@@ -41,22 +38,22 @@ public List<Car> getWinner() {
 
     public int getMaxLocation() {
         return cars.stream()
-                .mapToInt(car -> car.location)
+                .mapToInt(car -> car.getLocation())
```

Comment:
람다식을 사용할 때, 람다 함수의 시그니처가 사용할 함수의 시그니처와 일치하면, 다음과 같이 "메서드 참조"를 사용해서 더 간결하게 표현할 수 있어요.
```suggestion
                .mapToInt(Car::getLocation)
```